AAI/AAIConnectProfile.md

```diff
@@ -511,128 +503,169 @@ JWTs follow <https://tools.ietf.org/html/rfc7515> (JWS).
-     `aud`, `idp`, `scope`, `ga4gh_userinfo_claims`.
+#### Embedded Token issued by Embedded Token Signatory
+
+There are two supported formats for Embedded Tokens.
+
+##### Embedded Access Token Format
+
+Follows the [Broker access token](#access-token-issued-by-broker) format
+with the following caveats:
      
-2.   Follows the [Access Token](#access-token-issued-by-broker) format, except
-     it does contain a `jku` in the header and MUST NOT have "openid" in the
-     `scope` claim. When a `jku` is present, only the following headers and
-     claims are REQUIRED:
-
-     ```
-     {
-       "typ": "JWT",
-       "alg": "RS256",
-       "jku": "https://\<jwk URL\>",
-       "kid": "xxxxx"
-     }.
-     {
-       "iss": "https://\<issuer website\>/",
-       "sub": "\<subject identifier\>",
-       "iat": 1553545136,
-       "exp": 1553631536
-     }.
-     \<signature\>
-     ```
+1.  MUST NOT contain a `jku` in the header.
+
+2.  The payload claims MUST NOT include `aud`.
+
+3.  The payload claims MUST contain at least one GA4GH Claim.
+
+##### Embedded Document Token Format
+
+Conforms with JWS format requirements and is signed by an Embedded Claim
+Signatory.
+
+1. MUST be a JWS string.
+
+2. MUST contain a `jku` in the header.
+
+3. MUST NOT contain "openid" as a space-delimited substring of the `scope`
+   JWT claim, if the `scope` claim is provided.
+   
+4. The following headers and JWT claims in the payload are REQUIRED
+   (here shown in its decoded JSON form):
+
+   ```
+   {
+     "typ": "JWT",
+     "alg": "RS256",
+     "jku": "https://<jwk-URL>",
+     "kid": "<key-identifier>"
+   }.
+   {
+     "iss": "https://<issuer-website>/",
+     "sub": "<subject-identifier>",
+     "iat": <seconds-since-epoch>,
+     "exp": <seconds-since-epoch>,
+     <ga4gh-spec-claims>
+   }.
+   <signature>
+   ```
+
+   -   `typ`: MUST be "JWT".
+
+   -   `alg`: MUST be "RS256".
      
-     -   `jti` is RECOMMENDED.
+   -   `jti`: RECOMMENDED. A unique identifier for the token as per
+       [RFC7519 Section 4.1.7](https://tools.ietf.org/html/rfc7519#section-4.1.7)
+       is RECOMMENDED.
      
-     -   \<ga4gh-spec-claims\> MAY be present. A non-normative example is the
-         [ga4gh](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/RI_Claims_V1.md#example-ri-claims)
-         claim.
+   -   `<ga4gh-spec-claims>`: REQUIRED. One or more GA4GH Claims MUST be
+       provided. See [Authorization/Claims](#authorizationclaims) for an
+       example.
 
 #### Authorization/Claims 
 
 User attributes and claims are being developed in [GA4GH Researcher Identity
 Claims](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/RI_Claims_V1.md)
 document by the DURI work stream.
 
+A non-normative example of a GA4GH Researcher Identity Claim, as referred to
+in as `<ga4gh-spec-claims>` within the JWT formatting sections of this
+specification, is:
+
+```
+"ga4gh": {
```

Comment:
This example uses the "ga4gh" claim key for RI claims, but the RI Claims specification uses "ga4gh_passport_v1" key.

Reply:
Done